src/components/rocketsApi.js

```diff
@@ -0,0 +1,12 @@
+const rocketsUrl = 'https://api.spacexdata.com/v3/rockets';
+
+const fetchApiRockets = async () => {
+  const response = await fetch(rocketsUrl, {
+    method: 'GET',
+  });
+  const result = await response.json();
+  // console.log(result);
```

Comment:
- [ ] My comment about production code also applies here

Reply:
Thank you for your feedback.
All console.log statements have been removed.

Reply:
Done ✔️